src/Symfony/Component/Security/Core/User/UserInterface.php

```diff
@@ -73,6 +75,8 @@ public function getSalt();
      * Returns the username used to authenticate the user.
      *
      * @return string The username
+     *
+     * @deprecated since Symfony 5.3, use/implement getUserIdentifier() instead
```

Comment:
```suggestion
     * @deprecated since Symfony 5.3, use/implement getIdentifier() instead
```

it is `UserInterface`, rewriting `user` seems redondant, wdyt?

Reply:
This method will mostly be implemented by Doctrine-managed entities, naming it `getIdentifier()` might conflict. The only alternative I can think of is `getAuthIdentfier()`, but I'm fine with `getUserIdentifier()` personally.

Reply:
I agree with Robin here. Having a `getId()` and `getIdentifier()` is confusing, which is why I went with "user identifier". We can also make it e.g. security identifier, but I thought using "user" is the most descriptive.

Reply:
Understood :)